server/app/repository/CategoryRepository.java

```diff
@@ -0,0 +1,60 @@
+package repository;
+
+import com.google.common.collect.ImmutableList;
+import io.ebean.DB;
+import io.ebean.Database;
+import javax.inject.Inject;
+import models.CategoryModel;
+
+public final class CategoryRepository {
+
+  private static final QueryProfileLocationBuilder queryProfileLocationBuilder =
+      new QueryProfileLocationBuilder("CategoryRepository");
+
+  private final Database database;
+
+  @Inject
+  public CategoryRepository() {
+    this.database = DB.getDefault();
+  }
+
+  public ImmutableList<CategoryModel> findCategoriesByIds(ImmutableList<Long> ids) {
```

Comment:
**suggestion:** How do you anticipate this method will be used? I think we should only add methods to this file if we plan to use them soon, otherwise I think it's worth waiting to add them.

Reply:
That's a good consideration.  I'm planning on needing that method when I create the category checkbox options for when an admin creates or edits a program.  I intend to work on that as soon as I've added categories to the database seed tasks.  Do you think that's soon enough for me to leave it?

Reply:
Yeah, sounds good!! Just as long as there's an anticipated usage for it :) Thanks for confirming!